### PR TITLE
DJ January 2021

### PR DESCRIPTION
**Pull request by xaur**
**DRAFT**: https://github.com/xaur/decred-news/blob/draft202101/journal/202101.md

**Dependencies**: Send notice in advance so that people can plan their work.

- [ ] title image via an [issue in dcrdesign](https://github.com/decred/dcrdesign/issues) (notify by: 20th, ready by: 3rd)
- [ ] first time contributors
- [ ] dev activity stats
- [ ] PR update (notify by: 28th, ready by: 3rd)

**End of month snapshots**: Snapshot data that is hard to obtain later.

- [ ] social media stats
  - at the very minimum, snapshot stats reported in People
  - (opt) all accounts tracked by [social-media-stats](https://github.com/decredcommunity/social-media-stats)
  - Reddit mod stats
- [ ] mining: absolute and share% pool hashrate values
  - https://miningpoolstats.stream/decred
  - https://dcrstats.com/pow
- [ ] staking
  - 30-day average ticket price (https://dcrstats.com)
- [ ] LN
  - [LN map](https://ln-map.jamieholdstock.com/)

**Writing**

- [ ] extract more dev updates from GitHub repos
- [ ] complete Development (by: 2nd)
- [ ] complete People (by: 2nd)
- [ ] PO review Development and People
- [ ] ask in #dev to review Development and People
- [ ] complete Events
- [ ] (opt) ask #events to review Events
- [ ] complete Outreach
- [ ] complete Media
- [ ] (opt) ask #marketing to review Outreach+Events+Media
- [ ] complete Governance
- [ ] complete Network
- [ ] complete Integrations, Adoption
- [ ] (opt) find interesting Reddit and Twitter threads
- [ ] complete Community Discussions
- [ ] complete Markets
- [ ] complete Relevant External
- [ ] (opt) ask quoted people to review their texts
- [ ] complete Intro
- [ ] PO review of all sections
- [ ] (opt) ask #writers to review (by: 3rd)
- [ ] incorporate all feedback
- [ ] resolve all [TODOs](https://github.com/xaur/decred-news/blob/docs/guidelines.md#todos)
- [ ] ensure that all people who contributed or gave feedback are in [credits](https://github.com/xaur/decred-news/blob/docs/guidelines.md#how-to-give-credit)
- [ ] [lint](https://github.com/xaur/decred-news/blob/docs/guidelines.md#linting)
- [ ] add [title image](https://github.com/xaur/decred-news/blob/docs/guidelines.md#title-image)
- [ ] remove `{DRAFT}` from title
- [ ] pre-release review: check spelling, grammar, test that links work

**Publishing**

- [ ] publish on GitHub with a single commit to `gh-pages`
- [ ] publish on Medium
  - add to /decred/journals
  - add relevant tags
  - test short preview shown at /decred and /decred/journals
  - check for formatting issues
- [ ] publish on Publish0x
  - add relevant tags
- [ ] submit Medium link to r/decred
- [ ] tweet Medium link via decredproject. Don't forget to add title image. (by: 6th)
- [ ] submit new issue to decred.org Press page ([example](https://github.com/decred/dcrweb/pull/898))
- [ ] publish on CoinMarketCap
- [ ] if changes are necessary after the release, add commits to `gh-pages` (test in `draftXX` if needed) and update other places
- [ ] [update index](https://github.com/xaur/decred-news/blob/docs/guidelines.md#updating-index)
- [ ] fast-forward `master` to `gh-pages`

**Housekeeping**

- [ ] create archiving Git tag from the draft branch, e.g. `archive/draft1901`
- [ ] delete the draft branch
- [ ] (opt) update `journal-template.md` in `docs` branch
- [ ] (opt) update this `release-checklist-template.md` in the `docs` branch
- [ ] start new cycle

### CODE REVIEW

File: journal/202101.md [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+
+{ also check the base library https://github.com/planetdecred/dcrlibwallet }
+
+**[dcrios](https://github.com/planetdecred/dcrios)**
+
+- {}
+
+**[godcr](https://github.com/planetdecred/godcr)**
+
+- {}
```
[the remark this answers is not in the document]
Comment (xaur) [reply journal/202101.md]: @oshorefueled thanks for the info! I merged it with what I've written in parallel with you, please see if it reads well or if I should change anything.

**Comment by xaur** [comment]
Dear developers, Dev section for the upcoming January journal is ready for your review: https://github.com/xaur/decred-news/blob/draft202101/journal/202101.md#development

cc @davecgh @jrick @dnldd @alexlyp @matheusd @jholdstock @chappjc @lukebp @buck54321 @JoeGruffins @raedah @C-ollins

Feel free to hit Unsubscribe on the right sidebar as soon as you no longer need notifications from here.

**Comment by xaur** [comment]
@degeri @RichardRed0x FYI I have an issue connecting to Matrix that I hope to resolve soon.

**Comment by davecgh** [comment]
dcrd section looks fine.

**Comment by dnldd** [comment]
In addition, for dcrpool support for reverse proxy deployments is being added. NiceHash support is also in progress.

File: journal/202101.md [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+A total of 47 PRs from 10 contributors were [merged](https://github.com/decred/dcrd/pulls?q=is%3Apr+merged%3A2021-01-01..2021-01-31+sort%3Aupdated-asc), adding 12K and deleting 7K lines of code.
+
+In progress:
+
+- introduce [Age-Partitioned Bloom Filters](https://github.com/decred/dcrd/pull/2579) to significantly reduce memory used for tracking what data other peers are known to have
+
+**[dcrwallet](https://github.com/decred/dcrwallet)**
+
+- VSP client was [rewritten](https://github.com/decred/dcrwallet/pull/1956) to improve background processing and support recovery of fee payment information after seed restores { better translation to User-English is welcome, I didn't quite grasp the first design goal of 1956 }
+- narrow [rules](https://github.com/decred/dcrwallet/pull/1961) of what is considered a possible CoinJoin { I got what this change does but now where it is used }
```
Comment (jholdstock) [inline journal/202101.md:58]: Used for automatically detecting a wallets mixed account during recovery from seed.

File: journal/202101.md [hunk 3]
```diff
@@ -0,0 +1,376 @@
...
+- improved proposal [billing](https://github.com/decred/politeiagui/pull/2275) summary
+- improved [code stats](https://github.com/decred/politeiagui/pull/2279) display
+- bug fixes
+
+{ tlog update would be nice }
+
+**[vspd](https://github.com/decred/vspd)**
+
+- make sending of ticket's [parent](https://github.com/decred/vspd/pull/218) transaction mandatory as it greatly reduces the chance of ticket broadcast errors
+- update dependencies
```
Comment (jholdstock) [inline journal/202101.md:94]: Could mention that vspd version 1.0.0 is now tagged and all VSP operators should upgrade their mainnet deployment.

1.0.0 marks the beginning of a proper release process, where VSP operators should only ever run tagged releases in production (until now, vspd has still been in dev and operators have just been deploying whatever was latest master)

File: journal/202101.md (hunk 2 above)
Comment (xaur) [reply journal/202101.md]: Updated.

File: journal/202101.md (hunk 3 above)
Comment (xaur) [reply journal/202101.md:94]: I missed that, thanks! Added.

**Pull request closed without merging**